cmd/listener_windows.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 Mesosphere, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"net"
+	"os"
+)
+
+func getListener(unsetEnv bool) ([]net.Listener, error) {
+	return nil, nil
+}
+
+func getFiles(unsetEnv bool) []*os.File {
+	return nil
+}
```

Comment:
maybe instead of having 2 separate files like `listener_linux.go` and `listener_windows.go` we should define a simple `if linux then... else if windows then ...` ?

Reply:
That was what I tried in the first place. It won't compile because the go-systemd/activation package only exists in the Linux version of the Golang.